### ISSUE TITLE
Mobile virtual gamepad alignment

### ISSUE THREAD

**Issue by shaylevary** [caiiiycuk/js-dos.com#242]
Hi,

I would like to ask if there is an option to adjust the location of the virtual gamepad/nipple on screen?
Currently, when playing a fullscreen game In mobile, the virtual gamepad appears on top of the game screen, so I wonder if there is an option to set the virtual gamepad to appear on the left/right side the blank gaps of the screen, instead of on top of the game screen.

For example:
![Screenshot_20220808-001705_Chrome](https://user-images.githubusercontent.com/10597517/183311615-368a2e1f-0319-4316-9197-f903e21cea5c.jpg)

**Comment by caiiiycuk** [comment]
Don’t understand you. The nipple is appeared in place where it configured to be.

**Comment by shaylevary** [comment]
You are right, but it would be better if the nipple was displayed a little further to the right corner side of the screen (in the dead blank area), instead of displayed on a part of the game window in a way that hides some of the game screen.
